src/Components/Aspire.Azure.AI.OpenAI/Aspire.Azure.AI.OpenAI.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can you also update

https://github.com/dotnet/aspire/blob/main/src/Components/Aspire_Components_Progress.md
https://github.com/dotnet/aspire/blob/main/src/Components/Telemetry.md

?